### PR TITLE
adds centcomm jello

### PR DESCRIPTION
**Pull request by Plyushsune**
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+- type: entity
+  name: Centcomm jello
+  parent: FoodsnackBase
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml:3]:
```suggestion
  parent: FoodSnackBase
```

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  id: CentcommJello
+  description: Now 30 percent more air for 120 percent of the price
+  components:
+    - type: FlavorProfile
+      flavors:
+        - overpriced
+        - Jello
+    - type: Sprite
+      state: jello
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml:12]:
```suggestion
      state: Jello
```

File: Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    - type: SolutionContainerManager
+      solutions:
+        food:
+          maxVol: 1
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Objects/Consumable/Food/snack.yml:16]:
```suggestion
          maxVol: 3
```
The reagents that compose the jello exceed the capacity of the jello.

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
There's no way to obtain this in-game.

**Comment by toasterpm87** [comment]
> There's no way to obtain this in-game.

thats intentional

File: Resources/Textures/DeltaV/Objects/Consumable/Food/centcommjello.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "Adei",
```
Comment (DebugOk) [inline Resources/Textures/DeltaV/Objects/Consumable/Food/centcommjello.rsi/meta.json:4]: Adei? Adei who? From where?

**Review by DebugOk — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Objects/Consumable/Food/centcommjello.rsi/meta.json (hunk 4 above)
Comment (toasterpm87) [reply Resources/Textures/DeltaV/Objects/Consumable/Food/centcommjello.rsi/meta.json:4]: discord, she plays lukusha and valentin

**Comment by Colin-Tel** [comment]
Closing this for now.

**Pull request closed without merging**